.github/workflows/ci.yml

```diff
@@ -147,4 +147,19 @@ jobs:
           PROPERTY_PROVIDER: 'azure'
           RESOURCE_SNAPSHOT_CREATION_MINIMUM_INTERVAL: ${{ matrix.resource-snapshot-creation-minimum-interval }}
           RESOURCE_CHANGES_COLLECTION_DURATION: ${{ matrix.resource-changes-collection-duration }}
-
+      
+      - name: Collect logs
+        if: always()
+        run: |
+         make collect-e2e-logs
+        env:
+          KUBECONFIG: '/home/runner/.kube/config'
+          LOG_DIR: 'logs-${{ matrix.customized-settings }}'
+      
+      - name: Upload logs
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: e2e-logs-${{ matrix.customized-settings }}
```

Comment:
Each test run collects log in `test/e2e/logs-default(custom)/TIMESTAMP` subfolder. What is this `e2e-logs-*` name?

Reply:
After looking at the e2e run result, I realize this is the name of the uploaded zip file. Do we need a random suffix or timestamp to differentiate names across concurrent runs too?

Reply:
we don't need, cause each run will generate a new artifact ID, 

```
Artifact e2e-logs-default has been successfully uploaded! Final size is 1217630 bytes. Artifact ID is 3544533151
Artifact download URL: https://github.com/kubefleet-dev/kubefleet/actions/runs/16316461753/artifacts/3544533151
```
https://github.com/kubefleet-dev/kubefleet/actions/runs/16316461753/job/46083454280
https://github.com/kubefleet-dev/kubefleet/actions/runs/16316461753/job/46094038978